data/dataverses/dataverseno/datasets/tabular-sample-data/tabular-sample-data.json

```diff
@@ -1 +1,191 @@
+                    "typeName": "dateOfDeposit",
+                    "multiple": false,
+                    "typeClass": "primitive",
+                    "value": "2020-06-04"
+                }]
+            },
+            "geospatial": {
+                "displayName": "Geospatial Metadata",
+                "fields": []
+            },
+            "journal": {
+                "displayName": "Journal Metadata",
+                "fields": []
+            }
+        },
+        "files": [{
+            "description": "Excel file",
+            "label": "Tabular_Sample_Data.tab",
+            "restricted": false,
+            "version": 3,
+            "datasetVersionId": 134195,
+            "dataFile": {
+                "id": 1104456,
+                "persistentId": "doi:10.70122/FK2/0MVCBM/LISFFL",
+                "pidURL": "https://doi.org/10.70122/FK2/0MVCBM/LISFFL",
+                "filename": "Tabular_Sample_Data.tab",
+                "contentType": "text/tab-separated-values",
+                "filesize": 2200,
+                "description": "Excel file",
+                "storageIdentifier": "file://17282ef3127-d8ed28281ac7",
+                "originalFileFormat": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+                "originalFormatLabel": "MS Excel Spreadsheet",
+                "originalFileSize": 12751,
+                "UNF": "UNF:6:eFXLLKw9zMMIcjPRZOLhPg==",
+                "rootDataFileId": -1,
+                "md5": "45a5a81537c772485a640a030b12bd33",
+                "checksum": {
+                    "type": "MD5",
+                    "value": "45a5a81537c772485a640a030b12bd33"
+                },
+                "creationDate": "2020-06-05"
+            }
+        }, {
+            "description": "Tab-separated plain text file",
+            "label": "Tabular_Sample_Data.txt",
+            "restricted": false,
+            "version": 1,
+            "datasetVersionId": 134195,
+            "dataFile": {
+                "id": 1104457,
+                "persistentId": "doi:10.70122/FK2/0MVCBM/IYDYQR",
+                "pidURL": "https://doi.org/10.70122/FK2/0MVCBM/IYDYQR",
+                "filename": "Tabular_Sample_Data.txt",
+                "contentType": "text/plain",
+                "filesize": 2060,
+                "description": "Tab-separated plain text file",
+                "storageIdentifier": "file://17282ee8e22-382bc0edc2d6",
+                "rootDataFileId": -1,
+                "md5": "b2a1624bcf972094c48d639d4aa67c9e",
+                "checksum": {
+                    "type": "MD5",
+                    "value": "b2a1624bcf972094c48d639d4aa67c9e"
+                },
+                "creationDate": "2020-06-05"
+            }
+        }],
+        "citation": "Test User 1, UiT, 2020, \"Testing Tabular File Ingest - Excel and tab-separated .txt\", https://doi.org/10.70122/FK2/0MVCBM, Demo Dataverse, V1, UNF:6:eFXLLKw9zMMIcjPRZOLhPg== [fileUNF]"
+    }
+}
```

Comment:
Thanks for reformatting this. Oddly, the output says the dataset was published...

```
Publishing dataset id 3
<Response [200]>
```

... but when you log into the app you can see that publishing failed:

<img width="1055" alt="Screen Shot 2021-06-29 at 3 05 28 PM" src="https://user-images.githubusercontent.com/21006/123853506-88dc8680-d8eb-11eb-9a99-b05bfba60491.png">

I retried with curl and saw 403 instead of 200, which is good:

$ curl -i -H "X-Dataverse-key: $API_TOKEN" -X POST "$SERVER_URL/api/datasets/:persistentId/actions/:publish?persistentId=$PERSISTENT_ID&type=$MAJOR_OR_MINOR"
HTTP/1.1 403 Forbidden
Server: Payara Server  5.2020.6 #badassfish
X-Powered-By: Servlet/4.0 JSP/2.3 (Payara Server  5.2020.6 #badassfish Java/AdoptOpenJDK/11)
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: PUT, GET, POST, DELETE, OPTIONS
Access-Control-Allow-Headers: Accept, Content-Type, X-Dataverse-Key
Content-Type: application/json;charset=UTF-8
Content-Length: 162
X-Frame-Options: SAMEORIGIN

{"status":"ERROR","message":"This dataset cannot be published because some files have been found missing or corrupted. . Please contact support to address this."}